.github/workflows/main.yml

```diff
@@ -31,21 +31,25 @@ jobs:
       matrix:
         os: [ubuntu-latest, windows-latest, macos-14]
     steps:
-      - name: Set up repository
-        uses: tree-sitter/parser-setup-action@v1.2
-        with:
-          node-version: 20
+      - name: Clone repository
+        uses: actions/checkout@v4
       - name: Clone nvim help files
         uses: actions/checkout@v4
         with:
           repository: neovim/neovim
           path: examples/neovim
           sparse-checkout: runtime/doc/
-      - name: Run tests
+      - name: Set up tree-sitter
+        uses: tree-sitter/setup-action/cli@v1
+      - name: Run parser tests
         uses: tree-sitter/parser-test-action@v2
         with:
-          test-library: ${{runner.os == 'Linux'}}
-          corpus-files: |-
+          test-rust: ${{runner.os == 'Linux'}}
+      - name: Parse sample files
+        uses: tree-sitter/parse-action@v4
+        id: parse-files
+        with:
+          files: |-
             examples/neovim/runtime/doc/*
           # FIXME: these files should not have errors
```

Comment:
I think we fixed all these files; please try not excluding them.

Reply:
Ah, but those are not synced with the main repo. That probably should be done as well (but not in this PR).

Reply:
Ah, looks like this includes the parsing errors from injections (which core tests ignore). Still, the list looks like it has to be updated.

Reply:
All of the files except for one fail to parse on Windows.